cmd/sink/filters/filters.go

```diff
@@ -15,7 +15,8 @@ import (
 
 	"github.com/fsnotify/fsnotify"
 	"github.com/google/cel-go/cel"
-	"github.com/kubearchive/kubearchive/cmd/sink/k8s"
+	kubearchivev1alpha1 "github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1"
```

Comment:
This label includes the version "v1alpha1", are you going to change this name, and all the references, to something else when we release "beta" and "v2" code? It seems like it would make more sense to call this "kubearchiveapi", "operatorapi", or just "oapi" so when we change the operator version we only have to change one line in this file.

Reply:
Good point. I've done that.